backend/src/main/resources/db/migration/V2_1_1__checkInRefactoringDropAttrsAddNew.sql

```diff
@@ -0,0 +1,57 @@
+ALTER TABLE MEASURE
+    RENAME VALUE TO VALUE_METRIC;
+ALTER TABLE MEASURE
+    ALTER COLUMN VALUE_METRIC DROP NOT NULL,
+ALTER
+COLUMN MEASURE_DATE DROP
+NOT NULL,
+    ALTER
+COLUMN CHANGE_INFO DROP
+NOT NULL,
+    ALTER
+COLUMN CHANGE_INFO TYPE VARCHAR(4096),
+    ADD COLUMN CONFIDENCE    INTEGER,
+    ADD COLUMN CHECK_IN_TYPE VARCHAR(255),
+    ADD COLUMN ZONE TEXT;
+ALTER TABLE MEASURE
+    RENAME MEASURE_DATE TO MODIFIED_ON;
+
+UPDATE MEASURE
+SET CONFIDENCE = 5;
+UPDATE MEASURE
+SET CHECK_IN_TYPE = 'metric';
+
+ALTER TABLE "public"."measure"
+    RENAME TO "check_in";
+ALTER TABLE "public"."sequence_measure"
+    RENAME TO "sequence_check_in";
+
+DROP VIEW IF EXISTS OVERVIEW;
+CREATE VIEW OVERVIEW AS
+SELECT T.ID                AS "TEAM_ID",
```

Comment:
Die Aliase müssen lower case sein, sonst erhalte ich im Backend Fehler.
Wieso werden in dieser Datei grosse Feldnamen verwendet? Alle Felder in den Tabellen sind klein!